src/bin/common.rs

```diff
@@ -541,6 +559,13 @@ fn parse_config(matches: &ArgMatches<'_>) -> Result<EncoderConfig, CliError> {
     panic!("Tile columns and rows may not be greater than 64");
   }
 
+  if let Some(frame_rate) = matches.value_of("FRAME_RATE") {
+    cfg.time_base = Rational::new(
+      frame_rate.parse().unwrap(),
+      matches.value_of("TIME_SCALE").unwrap().parse().unwrap(),
```

Comment:
These two lines should be swapped. Both the ivf timebase and the sequence header timing info fields will have invalid values otherwise.

Reply:
Done